tests/CMakeLists.txt

```diff
@@ -27,9 +27,20 @@ if(REALM_ENABLE_SYNC)
     list(APPEND SOURCES sync.cpp)
 endif()
 
+find_package(Threads)
+find_package(PkgConfig REQUIRED)
+pkg_search_module(OPENSSL REQUIRED openssl)
```

Comment:
So it does, thank you for pointing it out. Cases like these are why we asked for help in the first place.